util/screenshots.js

```diff
@@ -58,7 +56,26 @@ export class Screenshots {
   }
 
   async takeThumbnail() {
-    await this.take("thumbnail");
+    try {
+      const screenshotType = "thumbnail";
+      await this.browser.setViewport(this.page, {width: 1920, height: 1080});
+      const options = screenshotTypes[screenshotType];
+      const screenshotBuffer = await this.page.screenshot(options);
+      const thumbnailBuffer = sharp(screenshotBuffer)
```

Comment:
I think this needs to have an await, as toBuffer returns a promise

Reply:
good catch, added